ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/MLChatAgentRunner.java

```diff
@@ -390,7 +404,7 @@ private void runReAct(
                                     .getMemoryManager()
                                     .updateInteraction(
                                         r.getId(),
-                                        ImmutableMap.of(AI_RESPONSE_FIELD, finalAnswer1),
+                                        ImmutableMap.of(AI_RESPONSE_FIELD, finalAnswer1, ADDITIONAL_INFO_FIELD, additionalInfo),
```

Comment:
We will also need to add additional info to memory as part of flow and react agent. Let's open an issue for the same.

Reply:
`ReAct` will be deprecated and replace by conversational